packages/@coorpacademy-components/src/molecule/brand-form-group/index.js

```diff
@@ -80,7 +80,17 @@ const buildInput = field => {
       return <BrandUploadBox {...field} />;
     case 'button':
     case 'link':
-      return <Button {...field} />;
+      return (
+        <ButtonLink
+          {...field}
+          type="primary"
```

Comment:
same here, the button background color will be modified

Reply:
same from above